web/projects/ui/src/app/pages/init/init.page.ts

```diff
@@ -1,11 +1,33 @@
-import { Component } from '@angular/core'
-import { ApiService } from 'src/app/services/api/embassy-api.service'
+import { Component, inject } from '@angular/core'
+import { Router } from '@angular/router'
+import { delay, filter } from 'rxjs'
+import { InitService } from 'src/app/pages/init/init.service'
 
 @Component({
   selector: 'init-page',
   templateUrl: 'init.page.html',
   styleUrls: ['init.page.scss'],
 })
 export class InitPage {
-  constructor(private readonly api: ApiService) {}
+  private readonly router = inject(Router)
+  readonly progress$ = inject(InitService)
+
+  readonly sub = this.progress$
+    .pipe(
+      filter(progress => progress === 1),
+      delay(500),
+    )
+    .subscribe(() => {
+      this.router.navigate([''])
+    })
+
+  getMessage(progress: number | null): string {
```

Comment:
Messaging here is not correct and unnecessary. Messaging will come from phase names, which should be returned from progress sub